run_analysis.R

```diff
@@ -1,6 +1,9 @@
 #HOMEWORK
 #Homework Getting and Cleanning Data
 
+# Make melt() function available
+library(reshape2)
```

Comment:
I needed to add this to avoid the following error:

```
Error in melt(dataset.filtered, id = c("activitylabel", "subject"), measure.vars = features.colnames) :
  could not find function "melt"
Execution halted
```